README.md

```diff
@@ -1,101 +1,140 @@
-The prerequisites for firebase cpp are required. Please install the following
-packages outlined in [README.md for firebase cpp](https://github.com/firebase/firebase-cpp-sdk/blob/master/README.md#prerequisites).
-
-The following prerequisites are required for all platforms.  Be sure to add any
-directories to your PATH as needed.
 
-- [CMake](https://cmake.org/) version 3.13.3, or newer.
-- [Mono](https://www.mono-project.com/) version 5 or newer.
-- [Unity](https://unity.com/) version 2019 or newer.
-- [Swig](http://www.swig.org/) version 4 or newer.
+Please follow [Firebase C++ SDK Prerequisites](https://github.com/firebase/firebase-cpp-sdk/blob/main/README.md#prerequisites) first.
 
-### Prerequisites for Windows
-On windows, to work around path length issues with google unity resolver enable
-long path support in git:
+- [Swig](https://www.swig.org/), version 4 or newer
 
-> **git config --system core.longpaths true**
+### Install Unity
 
-### Prerequisites for Mac
-Home brew can be used to install required dependencies:
+- [Unity](https://unity.com/download), version 2019 or newer
 
-```bash
-# https://github.com/protocolbuffers/protobuf/blob/master/kokoro/macos/prepare_build_macos_rc#L20
-ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-source $HOME/.rvm/scripts/rvm
-brew install cmake protobuf python mono swig
-sudo chown -R $(whoami) /usr/local
-```
+While installing through UnityHub, when you decide which version to install, it will pop up a dialog to select necessary support modules. Please check the boxes based on your dev machine OS and dev platform. For most common case while dev on macOS, we should select Android + Android SDK & NDK Tool, iOS Build Support and Mac Build Support (IL2CPP)
 
 ## Building
-The build uses CMake to generate the necessary build files, and supports out of
-source builds.
 
-The following CMake options are avaliable:
+Under the repo root folder, call
 
-* **FIREBASE_INCLUDE_UNITY**: Build for unity only (no mono support)
-* **FIREBASE_INCLUDE_MONO**: Build for mono only (no unity support)
-* **FIREBASE_UNI_LIBRARY**: Build all native modules as one dynamic lib
-* **FIREBASE_CPP_SDK_DIR**: Local path to firebase cpp
-* **FIREBASE_UNITY_SDK_VERSION**: Set version string of firebase unity package
-* **UNITY_ROOT_DIR**: Local path to Unity's installation directory
-  (path should end with version number)
-* **MONO_DIR**: Local path to mono's xbuild executable directory
-* **OPENSSL_ROOT_DIR**: Open ssl root directory
-* **PROTOBUF_SRC_ROOT_FOLDER**: Protobuf root directory
+``` bash
+python scripts/build_scripts/build_zips.py –platform=<target platform, pick one from linux,macos,windows,ios,android>
+```
 
-> Note:<br/>
-> &nbsp;&nbsp;&nbsp;**UNITY_ROOT_DIR** is a recommended setting for building
-> unity, else CMake will make an effort to auto find unity.
+Expected output artifact is
+[Repo Root]/<platform_unity, eg macos_unity>/firebase_unity-<version>-<platform>.zip
```

Comment:
version and platform don't seem to render correctly, might need to use something other than < and >

Reply:
fixed